save-common/src/commonMain/kotlin/org/cqfn/save/agent/TestExecutionDto.kt

```diff
@@ -8,14 +8,16 @@ import kotlinx.serialization.Serializable
  * @property id
  * @property agentId
  * @property status
- * @property startTime
- * @property endTime
+ * @property startTimeSeconds
+ * @property endTimeSeconds// todo: since datetime 0.2.0 can use serializable Instant
+ * @property filePath
  */
 @Serializable
 data class TestExecutionDto(
     val id: Long,
-    val agentId: Long,
+    val filePath: String,
```

Comment:
I thought about which information will be most useful from the user perspective. `agentId` is definitely not interesting, because it's our internal thing. And to distinguish these tests from each other (because they don't have names now) file path seemed to be the only good option

Reply:
But if we have path to test, can we extract just name of this test and show only it?

Reply:
What if there are tests with same name or, more probably, simply a lot of tests in different project modules? I think it would be more readable to show full path